### PR TITLE
rewrite your-first-algo.md

### PR DESCRIPTION
**Pull request by lemonez**
Basically a re-write. Much shorter now, removed unnecessary info. Some screenshots could use updating; will replace the appropriate files in that directory.

### CODE REVIEW

File: _pages/algorithm-development/your-first-algo.md [hunk 1]
```diff
@@ -13,216 +13,109 @@ redirect_from:
...
 
-Under the **Builds** tab you can view release notes and permissions associated with each published version of the algorithm.
+On Algorithmia, every algorithm is required to have an `apply()` function—it defines the algorithm's entry point and serves to standardize algorithms across the platform regardless of programming language. Authors are encouraged to leverage this standardization and to design their algorithms in a way that makes them predictable for end users and that supports algorithm pipelining (chaining multiple algorithms together).
 
-Under the **Versions** tab you can view information about all of the discreet algorithm builds that have occurred, including specific Git hashes and the version number associated with each.
+In order to run the algorithm, you'll first need to compile (build) it by clicking the "Build" button on the top-right corner of the source code editor. You'll need to complete this step even for an interpreted language such as Python, because we still need to package up all the source code into a callable algorithm instance on the back end.
```
Comment (bnlb) [inline _pages/algorithm-development/your-first-algo.md:76]: +1 for describing why this needs to happen 👏

File: _pages/algorithm-development/your-first-algo.md [hunk 2]
```diff
@@ -13,216 +13,109 @@ redirect_from:
...
+As you saw in this guide, you can create simple algorithms that don't read or write any data to disk. However, most algorithms you develop will need to be able to interact with some sort of data source.
 
-## Next Steps
+As an integration-first platform, we encourage you to connect to whatever data source(s) you're already using. We also enable you to host data directly on our platform for lower-latency access, and we even have several native integrationss with cloud-hosted data platforms.
 
-After you've finished this tutorial, you'll probably want to check out the <a href="{{site.baseurl}}/algorithm-development/languages">Language Guides</a> for how to write algorithms in the language you prefer, such as R, Python, Rust, Ruby, Java, Scala, or JavaScript.
+See our <a href="http://docs.algorithmia.com/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```
Comment (bnlb) [inline _pages/algorithm-development/your-first-algo.md:119]: This is an old url, can avoid a couple of directs by hitting the new url directly
```suggestion
See our <a href="https://algorithmia.com/developers/api/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```

**Review by bnlb — DISMISSED** [review]
✍️

File: _pages/algorithm-development/your-first-algo.md (hunk 2 above)
Comment (lemonez) [reply _pages/algorithm-development/your-first-algo.md:119]: 🙏🏼  thanks

File: _pages/algorithm-development/your-first-algo.md (hunk 1 above)
Comment (lemonez) [reply _pages/algorithm-development/your-first-algo.md:76]: 👍🏼

**Review by bnlb — APPROVED** [review]

**Pull request merged**